.github/workflows/ci.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ['9.4']
+        ghc: ['9.6']
```

Comment:
__question:__ Should the matrix also include 9.4 like the other change, or is there a reason to limit this to 9.6?

Reply:
Good catch